build.xml

```diff
@@ -958,4 +960,154 @@ ${git_status}
       </filterchain>
     </reflexive>
   </target>
+
+  <!-- Update bootstrap3 templates from bootstrap5 -->
+  <!-- N.B. This destroys any existing bootstrap5 templates (except explicitly excluded ones) -->
```

Comment:
"N.B. This destroys any existing _bootstrap3_ templates"?

Reply:
Fixed!